prac_08/convert_miles_km.py

```diff
@@ -0,0 +1,30 @@
+from kivy.app import App
+from kivy.app import Builder
+from kivy.properties import StringProperty
+
+MILES_TO_KM_CONVERSION = 1.60934
+
+
+class ConvertMiles(App):
+    kilometres = StringProperty()
+
+    def build(self):
+        self.title = "Convert Miles to Kilometres"
+        self.root = Builder.load_file("convert_miles_km.kv")
+
+    def handle_convert(self):
+        try:
+            conversion = float(self.root.ids.input_number.text) * MILES_TO_KM_CONVERSION
```

Comment:
Would it be more efficient to make self.root.ids.input_number.text into an argument (in the kv file) to pass as a parameter with a clearer name?